src/main/java/org/jabref/gui/ai/components/privacynotice/PrivacyNoticeComponent.fxml

```diff
@@ -0,0 +1,85 @@
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Text>
+            <Text layoutX="20.0" layoutY="77.0" strokeType="OUTSIDE" strokeWidth="0.0" text="%As soon as you ask a question, the text content of all PDFs attached to the entry are send to external service.">
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Text>
+            <HBox alignment="BASELINE_LEFT" spacing="5.0">
+               <children>
+                  <Text strokeType="OUTSIDE" strokeWidth="0.0" text="%If you have chosen the OpenAI as AI provider, the privacy policy of OpenAI applies. You find it at">
+                     <font>
+                        <Font size="14.0" />
+                     </font>
+                  </Text>
+                  <Hyperlink onAction="#onPrivacyHyperlinkClick" text="https://openai.com/policies/privacy-policy">
+                     <font>
+                        <Font size="14.0" />
+                     </font>
+                  </Hyperlink>
+               </children>
+            </HBox>
+            <HBox alignment="BASELINE_LEFT" spacing="5.0">
+               <children>
+                  <Text strokeType="OUTSIDE" strokeWidth="0.0" text="%If you have chosen the Mistral AI as AI provider, the privacy policy of Mistral AI applies. You find it at">
+                     <font>
+                        <Font size="14.0" />
+                     </font>
+                  </Text>
+                  <Hyperlink onAction="#onPrivacyHyperlinkClick" text="https://mistral.ai/terms/#privacy-policy">
+                     <font>
+                        <Font size="14.0" />
+                     </font>
+                  </Hyperlink>
+               </children>
+            </HBox>
+            <HBox alignment="BASELINE_LEFT" layoutX="20.0" layoutY="112.0" spacing="5.0">
+               <children>
+                  <Text strokeType="OUTSIDE" strokeWidth="0.0" text="%If you have chosen the Hugging Face as AI provider, the privacy policy of Hugging Face applies. You find it at">
+                     <font>
+                        <Font size="14.0" />
+                     </font>
+                  </Text>
+                  <Hyperlink onAction="#onPrivacyHyperlinkClick" text="https://huggingface.co/privacy">
+                     <font>
+                        <Font size="14.0" />
+                     </font>
+                  </Hyperlink>
+               </children>
+            </HBox>
+            <Text layoutX="20.0" layoutY="98.0" strokeType="OUTSIDE" strokeWidth="0.0" text="%Additionally, we use Deep Java Library for running embedding model. The embedding model will be downloaded in background from Deep Java Library servers.">
```

Comment:
1) I still haven't got an answer about their privacy policy. I'll ping them again

2) Oops, I use the same privacy policy for chatting and summarization. But in reality, summarization doesn't use embedding model.

I propose this:
- Let's say to users that the embedding model will be used for both chatting and summarization
- There are algorithms for generating summaries, that use embeddings
- The reason I want to say this, is because, maybe I refine the algorithm in future and it will really use embeddings?
- :heavy_plus_sign:  It's just easier to have one `PrivacyPolicyComponent.fxml` than two :smile: